tools/zbfind

```diff
@@ -1106,9 +1107,12 @@ class ZBPoller(Thread):
         except Exception, e:
             traceback.print_exc()
             sys.exit(1)
+        finally:
+            self.kb.sniffer_off()
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--iface', '--dev', action='store', dest='devstring', required=True)
```

Comment:
similarly should not be required and default to None

Reply:
as already pointed out: without specifying an interface, the tool wont work (for me):
```
python zbfind -c 11

Error initializing KillerBee Sniffer
```

Reply:
Yes, this is due to a specific thing where you are only using the Sewio sniffer, which doesn't support auto-discovery at all due to the API/etc.

Although this change makes perfect sense in your case, we need to ensure that the behavior users expect remains as in all the other devices, the auto-discovery works. We could consider changing this API at a major version, but can't change it at a minor version release as it would break the usage people expect.